java/vector/pom.xml

```diff
@@ -95,6 +95,18 @@
     </resources>
 
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <version>3.2.0</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>test-jar</goal>
```

Comment:
Just to clarify - so this packages vector tests in a jar that other modules can depend on?

Reply:
Actually, I think this is already declared in the root pom and these *-tests.jars already exist. So this isn't needed right?

Reply:
Right, removed.